Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -98,5 +97,21 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		Knights can use their swords to destroy wooden blocks and doors, but not stone;
 		Doors and siege weapons are vulnerable to explosives.;
 		Bison can be tamed with burgers.;
-		Resources can be resupplied by changing classes in workshops, not just at the tent!;
+		You can get resupplies in class workshops, not just at the tent!;
 		You can highlight useful items for your class by holding the pickup key.;
+		As a knight, you can always jab the enemy after successfully slashing them to deal extra heart of damage.;
+		You should always have one transport tunnel near your tent and another one at your tower for quick transportation.;
+		You should always have a working saw trap on your spawn tunnel.;
+		You can throw saws at logs to quickly turn them into usable wood.;
+		Knights can cut down trees quicker than builders. If your team has a tree farm, cut down trees as knight to save time for your builders.;
+		Double slashing deals damage faster than normal slashing or jabbing.;
+		A drill will overheat very quickly when drilling doors and platforms.;
+		Placing platforms inside walls doesn't help against explosions, no matter what direction they're facing.;
```

Comment:
In the future update it will work if placed the correct direction (they will be more blast resistant than a normal block). I might honestly just remove this one in general.

Reply:
If you make a "block platform block" wall and shoot it with a bomb arrow, platforms will take damage after bomb arrow hits the outer layer of the wall, same as in release. In test build they dont instantly break, but considering the fix to platforms blocking explosions, there will be no reason to place platforms instead of normal wooden blocks. 
Though this tip is targeted at people who use a specific old building meta that no longer works, so yeah, its better to just remove it.